django_renderpdf/helpers.py

```diff
@@ -94,9 +95,10 @@ def django_url_fetcher(url: str):
 
 def render_pdf(
     template: Union[List[str], str],
-    file_: IO,
+    file_: HttpResponse,
```

Comment:
This change is not correct; any file like can be used here (e.g.: I use this function to render files into an S3 bucket).